src/data/setoid/partition.lean

```diff
@@ -237,8 +244,28 @@ _ (subtype (@is_partition α)) _ (partial_order.to_preorder _) $ partition.order
 
 end partition
 
+section finpartition
```

Comment:
I don't think you're allowed to start a section inside a namespace then finish it outside the namespace, as you do here. It looks like you don't need the section anyway.

Reply:
There was a confusion in the `end` parts. (Bad copy-past of my part which I couldn't check because Lean is slow here.)
Now, everything is in the `section setoid`.